src/server/services/about/views/service.test.js

```diff
@@ -0,0 +1,214 @@
+import { nunjucksTestEnv } from '~/test-helpers/component-helpers.js'
+// eslint-disable-next-line n/no-unpublished-import
+import { toMatchFile } from 'jest-file-snapshot'
```

Comment:
rather than disabling this you can add it to the array in https://github.com/DEFRA/cdp-portal-frontend/blob/main/.eslintrc.cjs#L179